quandl/model/datatable.py

```diff
@@ -36,7 +36,6 @@ def download_file(self, file_or_folder_path, **options):
         while not file_is_ready:
             file_is_ready = self._request_file_info(file_or_folder_path, params=options)
             if not file_is_ready:
-                print(Message.LONG_GENERATION_TIME)
```

Comment:
This message is actually pretty useful. As generating a bulk download file may take a bit of time having this message be printed on repeat will help reassure users that the script is not hanging.

Ideally we replace these with Logger calls instead

Reply:
I feel like we should introduce that in another PR, which adequately introduces sane logging.

Reply:
Though if we can have a consensus sooner, it could be introduced in this series as a replacement.

Reply:
I've changed my mind; I'll introduce it in this series, but it's going to be lame.  I would have preferred we spend the time to introduce log.debug() in a few more places that make sense.  For now, we'll just replace print() with debug()